app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -287,6 +289,7 @@ private void initializeContacts() throws IOException {
     private void initializeUserdata() throws IOException {
         final File file = new File(profilesPath);
         if (!file.exists()) {
+            LOGGER.warn("Database didn't exist -> created a new one");
```

Comment:
Either move this log to the end of the if-statement or change `created a new one` to `creating a new one`. Also change `Database didn't exist` to `Userdata part of the database didn't exist` or move the log to the `initializeDatabase` method.